cmd/utils/flags.go

```diff
@@ -804,12 +804,50 @@ var (
 		Value:    "",
 		Category: flags.BuilderCategory,
 	}
+
+	// Builder rate limit settings
+
+	// BuilderRateLimitDuration determines rate limit of events processed by builder.
+	// Input is the time interval between events, which is converted to a rate.Limit
+	// which represents the number of events per second.
+	BuilderRateLimitDuration = &cli.StringFlag{
+		Name: "builder.rate_limit_duration",
+		Usage: "Determines rate limit of events processed by builder; a duration string is a possibly signed sequence " +
+			"of decimal numbers, each with optional fraction and a unit suffix, such as \"300ms\", \"-1.5h\" or \"2h45m\"",
+		EnvVars:  []string{"FLASHBOTS_BUILDER_RATE_LIMIT_DURATION"},
+		Value:    builder.RateLimitIntervalDefault.String(),
+		Category: flags.BuilderCategory,
+	}
+
+	// BuilderRateLimitMaxBurst determines the maximum number of events a builder can handle at any given point in time.
+	// The burst value can be thought of as a bucket of size b, initially full and refilled at rate r per second.
+	// b is defined by BuilderRateLimitMaxBurst and r is defined by BuilderRateLimitDuration.
+	// Additional details can be found on rate.Limiter documentation: https://pkg.go.dev/golang.org/x/time/rate#Limiter
+	BuilderRateLimitMaxBurst = &cli.IntFlag{
+		Name:     "builder.rate_limit_max_burst",
+		Usage:    "Determines the maximum number of burst events the builder can accommodate at any given point in time.",
+		EnvVars:  []string{"FLASHBOTS_BUILDER_RATE_LIMIT_MAX_BURST"},
+		Value:    builder.RateLimitBurstDefault,
+		Category: flags.BuilderCategory,
+	}
+
+	// BuilderBlockResubmitInterval determines the interval at which builder will resubmit block submissions.
+	BuilderBlockResubmitInterval = &cli.StringFlag{
+		Name:     "builder.block_resubmit_interval",
+		Usage:    "Determines the interval at which builder will resubmit block submissions",
+		EnvVars:  []string{"FLASHBOTS_BUILDER_RATE_LIMIT_RESUBMIT_INTERVAL"},
+		Value:    builder.BlockResubmitIntervalDefault.String(),
+		Category: flags.BuilderCategory,
+	}
+
 	BuilderEnableCancellations = &cli.BoolFlag{
 		Name:     "builder.cancellations",
 		Usage:    "Enable cancellations for the builder",
 		Category: flags.BuilderCategory,
 	}
+
 	// RPC settings
+
```

Comment:
Since those are not our flags, I'd probably suggest not to add this new line as a good measure to avoid rebase conflicts as much as possible

Reply:
fair point, updated